### PR TITLE
return running/active devices ONLY

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**

- [x] return running/active devices ONLY

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-327]

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run stage-mac
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [get running devices](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-7cf6beb7-f328-4a49-b5ff-e6dc66ce8cdb)

**_SCREENSHOTS?:_** 
Screenshot 2023-03-24 at 12 37 56

[AN-327]: https://airqoteam.atlassian.net/browse/AN-327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/device-registry/models/Event.js [hunk 1]
```diff
@@ -561,6 +563,44 @@ eventSchema.statics = {
         projection[as] = 0;
       }
 
+      if (running === "yes") {
```
Comment (Baalmart) [inline src/device-registry/models/Event.js:566]: entity logic to create projection for running devices only.

File: src/device-registry/routes/v1/events.js [hunk 2]
```diff
@@ -23,6 +23,127 @@ const headers = (req, res, next) => {
 router.use(headers);
 
 /******************* create-event use-case *******************************/
+router.get(
```
Comment (Baalmart) [inline src/device-registry/routes/v1/events.js:26]: route definition for running devices

File: src/device-registry/utils/generate-filter.js [hunk 3]
```diff
@@ -287,6 +288,10 @@ const generateFilter = {
         filter["day"]["$gte"] = generateDateFormatWithoutHrs(startTime);
       }
 
+      if (running) {
```
Comment (Baalmart) [inline src/device-registry/utils/generate-filter.js:291]: filter logic for running device

**Comment by Codebmk** [comment]
Thanks @Baalmart I've plugged this into collocation

**Pull request merged**